backend/app/pydantic_models/notes.py

```
@@ -1,11 +1,15 @@
 # do something here bruh
 from pydantic import BaseModel
-from typing import List
+from typing import List, Union
+from .question import Question
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)